### PR TITLE
Questbook grant proposals

### PR DESCRIPTION
**Pull request by valentinludu**
Create credentials for questbook approved proposals

### CODE REVIEW

File: lib/questbook/createProjectCredentials.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { log } from '@charmverse/core/log';
+import { prisma } from '@charmverse/core/prisma-client';
+import { getAddress } from 'viem';
+import { optimism } from 'viem/chains';
+
+import { signAndPublishCharmverseCredential } from 'lib/credentials/attestOffchain';
+import type { ExternalProjectMetadata } from 'lib/credentials/schemas/external';
+import { getProjectOwners } from 'lib/gitcoin/getProjectDetails';
```
Comment (motechFR) [inline lib/questbook/createProjectCredentials.ts:8]: Question; Does getProjectOwners only return safes and are all the projects owned by safes?

Looking at the code, it seems an EOA wallet would fall through and not be returned

File: lib/questbook/createProjectCredentials.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                  Source: 'Questbook',
+                  Event: 'Approved',
+                  GrantRound: application.grantTitle,
+                  Date: credentialDate,
+                  GrantURL: '',
+                  URL: application.proposalUrl
+                }
+              },
+              chainId: optimism.id,
+              recipient: owner
```
Comment (motechFR) [inline lib/questbook/createProjectCredentials.ts:70]: A unit test here where we mock external dependencies & API response would we good to confirm the call data

In particular, we want to ensure the owner is saved to lowercase

File: lib/questbook/createProjectCredentials.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+          try {
+            const cred = await signAndPublishCharmverseCredential({
+              credential: {
+                type: 'external',
+                data: {
+                  Name: application.projectName,
+                  ProjectId: externalProject.id,
+                  Source: 'Questbook',
+                  Event: 'Approved',
```
Comment (motechFR) [inline lib/questbook/createProjectCredentials.ts:62]: Suggestion; Rename to "Grant Approved" for better context

**Review by motechFR — APPROVED** [review]

File: lib/questbook/createProjectCredentials.ts (hunk 2 above)
Comment (valentinludu) [reply lib/questbook/createProjectCredentials.ts:70]: getSafeOwners returns all addresses in lowercase

File: lib/questbook/createProjectCredentials.ts (hunk 3 above)
Comment (valentinludu) [reply lib/questbook/createProjectCredentials.ts:62]: Done

**Pull request merged**